mod_form.php

```diff
@@ -323,6 +323,11 @@ public function definition() {
         ), null , get_string('auto_recording', 'zoom'));
         $mform->setDefault('auto_recording', $config->defaultautorecording);
 
+        //Enable mail notifications for teachers
```

Comment:
@nayanalinkstreet I feel this will be good if you bring the whole option below the remainder mail checkbox